### ISSUE TITLE
It plays really bad with vim-latexsuite

### ISSUE THREAD

**Issue by carldong** [begriffs/haskell-vim-now#106]
It seems that this bundle really disables all normal mode key bindings in vim-latexsuite, which makes it half less useful. I don't know which part does this, though

**Comment by begriffs** [comment]
When you run the haskell-vim-now installer it backs up your old vim config and puts a totally new one in its place. This new vim config won't include the tex suite, but here's how you can add it back.

Create a file called `~/.vim.local/plugins.vim` containing
```viml
Plug 'gerw/vim-latex-suite'
```

Then start vim and run `:PlugInstall`. You may have to restart vim, but you should be able to use the tex suite again.

**Comment by carldong** [comment]
No, I have it from distro package, and things like <++> works. Just any normal mode features fail (like`\ll`). If I go with `vim -u /etc/vimrc`, those features work again. So it has to be something about vim-now

**Comment by begriffs** [comment]
I can think of two more possible causes:

* `/etc/vimrc` contains mappings which are not being set when using another vimrc. Solution: `ln -s /etc/vimrc ~/.vimrc.local`. Haskell vim now will load whatever is in `~/.vimrc.local` after it has finished loading itself.
* HVN changes the leader key to `,`. So the command might be `,ll` rather than `\ll`. Again this might get solved by linking the vimrc.

**Comment by SimpleX91** [comment]
We'll add leader key override soon. So if you need that, wait a bit please.

**Comment by carldong** [comment]
latex-suite has both `,` leading and `\` leading commands. For example, `,li` is insert the list environment, `\ll` is compile.

**Comment by begriffs** [comment]
@carldong or @SX91 are either of you interested in sending a pull request to do leader override?
